packages/playground/remote/src/lib/is-uploaded-file-path.ts

```diff
@@ -1,9 +1,3 @@
 export const isUploadedFilePath = (path: string) => {
-	return (
-		path.startsWith('/wp-content/uploads/') ||
-		path.startsWith('/wp-content/plugins/') ||
-		path.startsWith('/wp-content/mu-plugins/') ||
-		path.startsWith('/wp-content/themes/') ||
-		path.startsWith('/wp-content/fonts/')
-	);
+	return /^wp-content\/(?:fonts|(?:mu-)?plugins|themes|uploads)\//.test(path);
```

Comment:
```suggestion
	return /^\/wp-content\/(?:fonts|(?:mu-)?plugins|themes|uploads)\//.test(path);
```
Small tweak: the path should include a leading forward slash.

Reply:
thanks! I've incorporated this into the patch